eng/packages/http-client-csharp-mgmt/package.json

```diff
@@ -39,6 +39,9 @@
   "dependencies": {
     "@azure-typespec/http-client-csharp": "1.0.0-alpha.20251215.3"
   },
+  "peerDependencies": {
+    "@typespec/http-client-csharp": "^1.0.0-alpha"
```

Comment:
I have concerns that the floating version number might cause indeterministic behaviors in the future, but we could keep this in mind and see what happens.